src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java

```diff
@@ -78,7 +78,7 @@ public boolean init(final Expression<?>[] exprs, final int matchedPattern, final
 	@Override
 	@Nullable
 	protected Entity[] get(final Event e) {
-		final Entity en = from == 0 ? EffSecSpawn.lastSpawned :  from == 1 ? EffShoot.lastSpawned : EffDrop.lastSpawned;
+		final Entity en = from == 0 ? EffSecSpawn.lastSpawned :  from == 1 ? EffSecShoot.lastSpawned : EffDrop.lastSpawned;
```

Comment:
Since you're editing it \:)
```suggestion
		final Entity en = from == 0 ? EffSecSpawn.lastSpawned : from == 1 ? EffSecShoot.lastSpawned : EffDrop.lastSpawned;
```